econml/inference.py

```diff
@@ -623,7 +633,9 @@ def summary_frame(self, alpha=0.1, value=0, decimals=3, feat_name=None, output_n
             The output dataframe includes point estimate, standard error, z score, p value and confidence intervals
             of the estimated metric of each treatment on each outcome for each sample X[i]
         """
-
+        feat_name = self.feat_name if feat_name is None else feat_name
+        treatment_name = self.treatment_name if treatment_name is None else treatment_name
+        output_name = self.output_name if output_name is None else output_name
```

Comment:
I think a more pythonic way to write this is

```suggestion
        # Assign names from dataframes if not provided as arguments
        feat_name = feat_name or self.feat_name
        treatment_name = treatment_name or self.treatment_name
        output_name = output_name or self.output_name
```

(although using `or` with non-booleans still seems a bit weird to me)

Reply:
Great suggestions. I am changing this everywhere. the changes are out of date, but will address in a separate commit.

Reply:
I actually realized this doesn't work with `numpy` arrays as they you can't evaluate the statement `if array`. Reverting to the previous solution.